text/0000-dynamic-attributes.md

```diff
@@ -0,0 +1,189 @@
+---
+title: `lwc:spread` Directive
+status: DRAFTED
+created_at: 2021-07-05
+updated_at: YYYY-MM-DD
+champion: Mohammed Abdul Sattar (abdulsattar)
+pr: https://github.com/salesforce/lwc-rfcs/pull/52
+---
+
+# Passing Dynamic Attributes/Properties
+
+## Summary
+This proposal adds a mechanism to pass a dynamic attributes/properties/listeners to elements in
```

Comment:
why distinguishing or mentioning attributes and listeners here? the way I see it... we only give them support for properties... and it is the responsibility of the developer to know what props to use, we don't try to make sense of them.